lib/components/Editor/CustomExtensions/Mention/index.js

```diff
@@ -0,0 +1,78 @@
+import React, { useRef } from "react";
+import { Email } from "@bigbinary/neeto-icons";
+
+import Avatar from "components/Common/Avatar";
+import Dropdown from "components/Common/Dropdown";
+
+import {
+  MENU_ICON_SIZE,
+  ICON_COLOR_ACTIVE,
+  ICON_COLOR_INACTIVE,
+} from "../FixedMenu/constants";
+
+const Mentions = ({ editor, mentions, showImageInMention }) => {
+  const dropdownRef = useRef();
+
+  const handleSelectedMention = (label) => {
+    editor
+      .chain()
+      .focus()
+      .insertContent([
```

Comment:
Instead of calling insertContent directly here, It is advisable to create custom command option for adding mentions. This is how other extensions in tiptap normally works. Please refer Variable extension for a already created custom command.